### DISCUSSION TITLE
分享一個酷酷的機械手臂

### DISCUSSION THREAD

**Discussion by brian431** [1101-GS3073-C/Python-Community-GS3073-C#463] [General]
用了非常多的技術，看來作者在電子、電機、資工，機械等方面皆有涉獵。不過我個人是一個都看不懂。
其中有機器學習也會使用到的矩陣運算。
https://b23.tv/OMPr37e

**Comment by Chenszuhong** [comment]
太難了吧，等強一點想要玩玩看

**Comment by BingRong1217** [comment]
好多機器學習都有用到數學

**Comment by Ben6uo** [comment]
雖然有關於機械手臂的專題 但感覺太難了

**Comment by limitx0** [comment]
雖然我看不懂，但是我大受震撼
這就是神仙和渺小的凡人的差距啊

**Comment by skyling0299** [comment]
機械手臂是具有模仿人類手臂功能並可完成各種作業的自動控制設備，這種機器人系統有多關節連結並節允許在平面或三度空間進行運動或使用線性位移移動。構造上由機械主體、控制器、伺服機構和感應器所組成，並由程式根據作業需求設定其一定的指定動作。感覺蠻有趣的，記得中原大學有類似的機械手臂學程，很特別~

**Comment by Raywei0214** [comment]
真的很酷，我以前就是看到這類酷東西才想要往這方面發展的

**Comment by Yilin20021016** [comment]
機械手臀是一個很複雜的東西 需要很多時間去了解

**Comment by awkward-willy** [comment]
不論是機器學習、人工智慧、機器手臂的控制，背後都是滿滿的數學與科學原理。若只是使用簡單的功能，網路上有許多套件等能達到快速建立的效果，但實際深入學習時真的是一個無底洞，需要花很多精力去學習，要有興趣才能精通相關知識。
